components/provisioner/assets/database/provisioner.sql

```diff
@@ -0,0 +1,91 @@
+-- Cluster
+
+CREATE TABLE cluster
+(
+    ID uuid PRIMARY KEY CHECK (id <> '00000000-0000-0000-0000-000000000000'),
```

Comment:
I would go with lowercase to be consistent with the rest.

```suggestion
    id uuid PRIMARY KEY CHECK (id <> '00000000-0000-0000-0000-000000000000'),
```